<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>1095</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="men">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>521</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>1</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>64</number>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Номер</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Возраст</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Вес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Рост</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Жел. возр.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Жел. вес</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Жел. рост</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Men</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Women</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пол:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isM">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isW">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ж</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имя:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addB">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>290</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Номер:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numSpin">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Личные:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предпочитаемые:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>81</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Возраст:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вес:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Рост:</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="ageSpin">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="weiSpin">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="heiSpin">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="lAge">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="rAge">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>200</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="lWei">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="rWei">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="lHei">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="rHei">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>200</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>230</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>260</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="sortB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сортировать по возрасту</string>
    </property>
   </widget>
   <widget class="QPushButton" name="candidateB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>391</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вывести всех подходящих для кандидата под номером:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="choiceSpin">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>390</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="allB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>371</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вывести все возможные пары:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="result">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>391</width>
      <height>391</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>376</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>26</number>
    </attribute>
    <column>
     <property name="text">
      <string>Пары</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="deleteB">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openB">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="women">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>531</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>1</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>64</number>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Номер</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Возраст</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Вес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Рост</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Жел. возр.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Жел. вес</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Жел. рост</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(запись будет перезаписана, если человек с введенным номером уже добавлен )</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1208</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
